src/box/box.cc

```diff
@@ -2727,20 +2745,214 @@ box_set_prepared_stmt_cache_size(void)
 	return 0;
 }
 
+/**
+ * Report crash information to the feedback daemon
+ * (ie send it to feedback daemon).
+ */
+static int
+box_feedback_report_crash(struct crash_info *cinfo)
+{
+	/*
+	 * Update to a new number if the format get changed.
+	 */
+	const int crashinfo_version = 1;
+
+	char *p = (char *)static_alloc(SMALL_STATIC_SIZE);
+	char *tail = &p[SMALL_STATIC_SIZE];
+	char *e = &p[SMALL_STATIC_SIZE];
+	char *head = p;
+
+	int total = 0;
+	(void)total;
+	int size = 0;
+
+#define snprintf_safe(...) SNPRINT(total, snprintf, p, size, __VA_ARGS__)
+#define jnprintf_safe(str) SNPRINT(total, json_escape, p, size, str)
```

Comment:
Nit: Let's undef these two macros as soon as we don't need them anymore.

Reply:
Done.